cdap-client-tests/src/test/java/co/cask/cdap/client/ApplicationClientTestRun.java

```diff
@@ -77,64 +80,71 @@ public void cleanup() throws Throwable {
+    Assert.assertEquals(1, appClient.list(namespace).size());
 
     try {
       // check program list
       LOG.info("Checking program list for app");
-      Map<ProgramType, List<ProgramRecord>> programs = appClient.listProgramsByType(FakeApp.NAME);
+      Map<ProgramType, List<ProgramRecord>> programs = appClient.listProgramsByType(app);
       verifyProgramNames(FakeApp.FLOWS, programs.get(ProgramType.FLOW));
       verifyProgramNames(FakeApp.MAPREDUCES, programs.get(ProgramType.MAPREDUCE));
       verifyProgramNames(FakeApp.WORKFLOWS, programs.get(ProgramType.WORKFLOW));
       verifyProgramNames(FakeApp.SERVICES, programs.get(ProgramType.SERVICE));
 
-      verifyProgramNames(FakeApp.FLOWS, appClient.listPrograms(FakeApp.NAME, ProgramType.FLOW));
-      verifyProgramNames(FakeApp.MAPREDUCES, appClient.listPrograms(FakeApp.NAME, ProgramType.MAPREDUCE));
-      verifyProgramNames(FakeApp.WORKFLOWS, appClient.listPrograms(FakeApp.NAME, ProgramType.WORKFLOW));
-      verifyProgramNames(FakeApp.SERVICES, appClient.listPrograms(FakeApp.NAME, ProgramType.SERVICE));
+      verifyProgramNames(FakeApp.FLOWS, appClient.listPrograms(app, ProgramType.FLOW));
+      verifyProgramNames(FakeApp.MAPREDUCES, appClient.listPrograms(app, ProgramType.MAPREDUCE));
+      verifyProgramNames(FakeApp.WORKFLOWS, appClient.listPrograms(app, ProgramType.WORKFLOW));
+      verifyProgramNames(FakeApp.SERVICES, appClient.listPrograms(app, ProgramType.SERVICE));
 
-      verifyProgramNames(FakeApp.FLOWS, appClient.listAllPrograms(ProgramType.FLOW));
-      verifyProgramNames(FakeApp.MAPREDUCES, appClient.listAllPrograms(ProgramType.MAPREDUCE));
-      verifyProgramNames(FakeApp.WORKFLOWS, appClient.listAllPrograms(ProgramType.WORKFLOW));
-      verifyProgramNames(FakeApp.SERVICES, appClient.listAllPrograms(ProgramType.SERVICE));
+      verifyProgramNames(FakeApp.FLOWS, appClient.listAllPrograms(namespace, ProgramType.FLOW));
+      verifyProgramNames(FakeApp.MAPREDUCES, appClient.listAllPrograms(namespace, ProgramType.MAPREDUCE));
+      verifyProgramNames(FakeApp.WORKFLOWS, appClient.listAllPrograms(namespace, ProgramType.WORKFLOW));
+      verifyProgramNames(FakeApp.SERVICES, appClient.listAllPrograms(namespace, ProgramType.SERVICE));
 
-      verifyProgramRecords(FakeApp.ALL_PROGRAMS, appClient.listAllPrograms());
+      verifyProgramRecords(FakeApp.ALL_PROGRAMS, appClient.listAllPrograms(namespace));
     } finally {
       // delete app
       LOG.info("Deleting app");
-      appClient.delete(FakeApp.NAME);
-      appClient.waitForDeleted(FakeApp.NAME, 30, TimeUnit.SECONDS);
-      Assert.assertEquals(0, appClient.list().size());
+      appClient.delete(app);
+      appClient.waitForDeleted(app, 30, TimeUnit.SECONDS);
+      Assert.assertEquals(0, appClient.list(namespace).size());
     }
   }
 
   @Test
   public void testDeleteAll() throws Exception {
-    Assert.assertEquals(0, appClient.list().size());
+    Id.Namespace namespace = Id.Namespace.DEFAULT;
```

Comment:
same here. Although this, like the previous comments about similar things are nits. I just think assigning a variable to a constant is unnecessary. Although I can see that doing it this way makes the variable name shorter.